broker/src/main/java/io/camunda/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -258,28 +256,31 @@ public void onUnrecoverableFailure() {
     actor.run(this::handleUnrecoverableFailure);
   }
 
-  private void onInstallFailure(final long term, final Throwable error) {
+  private void onInstallFailure(final Throwable error) {
     if (error instanceof UnrecoverableException) {
       LOG.error(
           "Failed to install partition {} with unrecoverable failure: ",
           context.getPartitionId(),
           error);
```

Comment:
I assume it would be quite interesting to log the current term and index right?

Reply:
Term might be interesting. But why should we log index? I assume you mean raft log index.

Reply:
Actually I meant what you added to the context, so term and role, sorry for the confusion. I think it makes sense to add more context.

Reply:
That make sense. Before we didn't know which role failed. But now it is in the context. So we should log it.